scripts/airtable-to-json.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config()
+const fs = require('fs')
+
+const Airtable = require('airtable')
+
+Airtable.configure({
+  endpointUrl: 'https://api.airtable.com',
+  apiKey: process.env.AIRTABLE_API_KEY,
+})
+const base = Airtable.base(process.env.AIRTABLE_APP_ID)
+
+const recs = []
+base('GeorgeFloyd')
+  .select({
+    view: 'Grid view',
+    // maxRecords: 10, // Omit for unlimited
+  })
+  .eachPage(
+    function page(records, fetchNextPage) {
+      records.forEach((record) => {
+        const f = record.fields
+        recs.push({
+          id: f['TGD number'] || null,
+          city: f.City || 'unknown',
+          state: f.State || '??',
```

Comment:
Let's use `null` instead of these arbitrary strings. That makes it easier to tell if there is a value in these fields or not.